# Nuclear_Cytoplasmic_Interactions
The all data in this paper could be found in https://zenodo.org/record/13924464

If you have any questions, please contact me at email houting2001@163.com. I am happy to hear from you.

paper:

MT_NU_GWAS: GWAS_GEMMA.worksteps, the workflow to perform GWAS base on nuclear and mitochondrial genomes; heatmap.R, mapping script of heatmap of significant associated variation sites in the nucleus and mitochondria between wild and cultivated grapes; heatmap_data.xlsx, the input file for drawing heatmap.

NUMT_NUPT: GO.R, the visualization script of gene ontology enrichment analysis results; Identify_NUMT.worksteps, the workflow to identify NUMTs; Identify_NUPT.worksteps, the workflow to identify NUPTs; KEGG.R, the visualization script of kyoto encyclopedia of gene and genome enrichment analysis results; NUPT_classification_stacking_diagram.R, the drawing script of NUPT sequence classification; distribution.R, the NUMTs/NUPTs distribution map script on the nuclear genome; rm_overlap.py--python script for remove the overlap area.

gene_content.R--the script that draw the gene content of 34 mitogenomes.

gene_content.xlsx--the result file of 34 mitochondrial genome of gene content.
